### ISSUE TITLE
Optimize slow get_organizations endpoint

### ISSUE THREAD

**Issue by nicholasserra** [SEED-platform/seed#747]
The get_organizations endpoint at `app/accounts/get_organizations/` takes 3 minutes if you're a superuser on seedtest.

**Comment by nicholasserra** [comment]
Going to wait on a seedtest data dump to fix this locally. cc @mmclark @RDmitchell

**Comment by mmclark** [comment]
Shared via gdrive

**Comment by nicholasserra** [comment]
Seems "sub_orgs" from the get_organizations payload isnt used anywhere in the app.

**Comment by nicholasserra** [comment]
I lied, it's used, but in a weird way.

**Comment by RDmitchell** [comment]
I don't know how things are set up in the code, but Sub Orgs do exist as a concept and should have specific functionality, even if the program isn't doing it right now. 

Maybe in the software structure, there is no difference between an org and a sub org, and it is just a matter of what organizations are associated with each other (?) that would make them the Parent or the Sub Org (?)

I wrote this doc (in 2014) when I was trying to figure out how Organizations worked (based on the state of SEED at that time), including the Parent/Sub Orgs relationship, which I think doesn't have much content in it. worked in the version of SEED that existed then. 

https://drive.google.com/a/lbl.gov/file/d/0B3fTKpZ9Dx7LOHpfanV0VktyN0U/view?usp=sharing

I want to revisit the Parent/Sub Org feature in March or April to see what is currently implemented in the program, what we think the feature should really be and how to get it to that point.

**Comment by RDmitchell** [comment]
Might be worth documenting, possibly in this issue, or in a new issue (?), how it is currently used?

**Comment by nicholasserra** [comment]
For the purposes of this ticket, no sub org stuff was changed.

**Comment by nicholasserra** [comment]
Reassigning for merge and testing.

**Comment by RDmitchell** [comment]
took slightly over a minute to get the list of organizations.
